src/Service/CallService.php

```diff
@@ -583,17 +583,28 @@ public function decodeResponse(
             return [];
         }
 
-        $this->callLogger->debug('Response content: '.$responseBody);
+        // This if is statement prevents binary code from being used a string.
+        if ($contentType !== 'application/pdf') {
+            $this->callLogger->debug('Response content: '.$responseBody);
+        }
```

Comment:
I do not understand what this if statement is for, could you explain?
I think you should/could probably do a check on $responseBody if that is the problem here?